jmsSource/src/test/java/io/vantiq/extsrc/jmsSource/TestJMS.java

```diff
@@ -436,6 +436,8 @@ public void testOddballJMSMessageTypes() {
     
     @Test
     public void testIncorrectOddballJMSTypes() {
+        checkAllJMSProperties(false);
+
```

Comment:
With this missing, test attempted to use things no defined.  This allows the test to run clean when no properly defined (test is simply skipped -- this is by design).